fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvManager.java

```diff
@@ -187,14 +201,18 @@ public Optional<KvTablet> getKv(TableBucket tableBucket) {
         return Optional.ofNullable(currentKvs.get(tableBucket));
     }
 
-    public void dropKv(TableBucket tableBucket) {
+    public void dropKv(
+            PhysicalTablePath physicalTablePath, TableBucket tableBucket, boolean deleteRemote) {
         KvTablet dropKvTablet =
                 inLock(tabletCreationOrDeletionLock, () -> currentKvs.remove(tableBucket));
 
         if (dropKvTablet != null) {
             TablePath tablePath = dropKvTablet.getTablePath();
             try {
                 dropKvTablet.drop();
+                if (deleteRemote) {
+                    dropRemoteKvSnapshot(physicalTablePath, tableBucket);
```

Comment:
This removal is very similar to remote log removal. We can move this invoking `KvManager.dropRemoteKvSnapshot` into `ReplicaManager#stopReplica` after `remoteLogManager.stopReplica`. Ideally, we should delete the remote files (log&kv) asynchronously to not block the worker thread. We can refactor them in the near future.

Reply:
Yes,i have notice that，I find RemoteLogManager to manager remote files of LogTable without RemoteKvManager to manager snapshot files of KvTable，is there any plan  to unify those for managering remote files? I will continue to pay attention about it and look forward to participating it.